nicoRDL/dotTheory-JS
# JS-401 (JS-357) Node.js Basics: Building a Command Line Application

## Related Ticket
https://codeandtheory.atlassian.net/browse/JS-357

## Description
- completes building a basic Node.js command line application

## Steps to Validate
- run the responseBodyBasics.js file in the terminal
- a JSON response body should be logged in the terminal pointing to the user named Chalker

## Risks
n/a
